### PR TITLE
feat(wrw): add support for bsc and tbsc

### PR DESCRIPTION
**Pull request by venkateshv1266**
[COIN-1403](https://bitgoinc.atlassian.net/browse/COIN-1403)

Working fine for HOT, COLD and CUSTODIAL wallets.

[COIN-1403]: https://bitgoinc.atlassian.net/browse/COIN-1403?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/helpers/index.ts [hunk 1]
```diff
@@ -231,3 +231,14 @@ export function updateKeysFromIdsWithToken<TParams extends UpdateKeysFromsIdsDef
     return updateKeysFromIds(coin, ...rest);
   }
 }
+
+export function getEip1559Params(coin: string, maxFeePerGas: number, maxPriorityFeePerGas: number) {
+  // bsc and tbsc doesn't support EIP-1559
+  if (coin === 'bsc' || coin === 'tbsc') {
+    return undefined;
```
Comment (mullapudipruthvik) [inline src/helpers/index.ts:238]: you need to take it from user form, how is it building the tx with `undefined` value ?

Comment (venkateshv1266) [reply src/helpers/index.ts:238]: If `eip1559 : undefined`, in the sdk we set the default gas price. [Reference Link](https://github.com/BitGo/BitGoJS/blob/8b13b167463533a051143374b9fea8ccc1ad63b9/modules/abstract-eth/src/abstractEthLikeNewCoins.ts#L1308)

File: src/helpers/config.ts [hunk 2]
```diff
@@ -792,6 +794,12 @@ export const evmCCRIntendedChainCoins: Record<string, readonly CoinMetadata[]> =
       allCoinMetas.eth,
       allCoinMetas.polygon,
     ] as const,
+    bsc: [
+      allCoinMetas.eth,
+    ] as const,
+    tbsc: [
+      allCoinMetas.hteth,
+    ] as const,
```
Comment (kamleshmugdiya) [inline src/helpers/config.ts:802]: add arb and opt as well for v4.

**Review by kamleshmugdiya — CHANGES_REQUESTED** [review]

File: src/helpers/config.ts (hunk 2 above)
Comment (venkateshv1266) [reply src/helpers/config.ts]: Added

**Review by kamleshmugdiya — APPROVED** [review]

**Pull request merged**